src/source/mod.rs

```diff
@@ -232,6 +234,20 @@ where
         amplify::amplify(self, value)
     }
 
+    /// Applies automatic gain control to the sound.
```

Comment:
I usually copy the documentation from the the source here (in this case agc::automatic_gain_control). This is what most users will see so the docs should be as extensive as at the source.

Reply:
Thanks i added docs there